### PR TITLE
Rotating navigation animation Day3 completed

### PR DESCRIPTION
**Pull request by PranavGPR**

### CODE REVIEW

File: src/pages/rot_nav.tsx [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+`;
+
+const Circle = styled.div`
+  position: fixed;
+  top: -10rem;
+  left: -10rem;
+  z-index: 1000;
+`;
+
+const CircleContainer = styled.div<{}>`
```
Comment (KrishnaMoorthy12) [inline src/pages/rot_nav.tsx:148]: You don't have to mention empty params

File: src/pages/rot_nav.tsx [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+  width: 20rem;
+  height: 20rem;
+  border-radius: 50%;
+  position: relative;
+  background-color: ${colors.primary};
+  transition: all 1s;
+`;
+
+const BarButton = styled.button<{}>`
```
Comment (KrishnaMoorthy12) [inline src/pages/rot_nav.tsx:157]: Here also

File: src/pages/rot_nav.tsx [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+  cursor: pointer;
+  background-color: transparent;
+  outline: none;
```
Comment (KrishnaMoorthy12) [inline src/pages/rot_nav.tsx:160]: I suggest removing these here and adding it in the bottom.

File: src/pages/rot_nav.tsx (hunk 2 above)
Comment (KrishnaMoorthy12) [reply src/pages/rot_nav.tsx:157]: Although, I'll merge. Because hey! it works.

**Review by KrishnaMoorthy12 — APPROVED** [review]

**Comment by KrishnaMoorthy12** [comment]
I also recommend changing the file name. But we can make these changes on the go.

**Comment by PranavGPR** [comment]
@KrishnaMoorthy12 I'll change all those you said and re-request you for reviewing

**Review by KrishnaMoorthy12 — APPROVED** [review]
Very good.

**Pull request merged**